mycroft/api/__init__.py

```diff
@@ -39,8 +39,8 @@ class InternetDown(RequestException):
     pass
 
 
-class Api(object):
-    """ Generic object to wrap web APIs """
+class Api():
```

Comment:
We might as well exclude the extra parentheses here as well (so it becomes `class Api:`).

Reply:
Ah, right you are!

Should be fixed now.